lisa/node.py

```diff
@@ -377,13 +428,15 @@ def from_remote(
 
         node = Node.create(
             len(self._list),
+            base_log_path=base_log_path,
             capability=node_runbook.capability,
-            node_type=node_runbook.type,
             is_default=node_runbook.is_default,
             logger_name=logger_name,
-            base_log_path=base_log_path,
+            name=node_runbook.name,
+            node_type=node_runbook.type,
+            sut=node_runbook.sut,
         )
-        self._list.append(node)
+        self._insert(node)
```

Comment:
The order should be kept.

Reply:
I can leave this out, I think I'll just override the iterator to skip non-sut nodes. Think the part
```
        for index, current_req in enumerate(self.nodes):
            if len(capability.nodes) == 1:
                current_cap = capability.nodes[0]
            else:
                current_cap = capability.nodes[index]
```
still being able to do its thing without worries.

Reply:
I guess I meant
```
    def initialize(self) -> None:
        for node in self._list:
```
not that one (misnomer). Anyway, I'll ditch the attempts to hide the listing, just making init on them different should be enough.